### ISSUE TITLE
x/benchmarks/driver: update for Go 1.5 runtime/trace

### ISSUE THREAD

**Issue by nickng** [golang/go#12343]
# Environment

Output from `go version`

```
    go version go1.5 linux/amd64
```
# Problem

The problem is related to compilation of `golang/benchmarks` (i.e. `golang.org/x/benchmarks`). 

I tried to run `go build golang.org/x/benchmarks/bench` (then go install) as suggested by README, to build the benchmark executable which runs the benchmark.

[Here](https://github.com/golang/benchmarks/blob/master/driver/driver.go#L91) suggests the existence of `StartTrace` and `StopTrace` in  `runtime/pprof` but they do not exist.
# What I expect to see

`bench` executable created in `$GOPATH/bin/` (no output)
# What I see instead

```
# golang.org/x/benchmarks/driver
driver/driver.go:91: undefined: pprof.StartTrace
driver/driver.go:95: undefined: pprof.StopTrace
```

**Comment by minux** [comment]
Those has been moved to runtime/trace package.
We should update x/benchmarks instead.

**Comment by ALTree** [comment]
This was fixed in golang/benchmarks@7e3f3ecec1321db81dac2b188f22839ec200f2ba. Closing here.
